scripts/exglobal_stage_ic.sh

```diff
@@ -66,7 +89,7 @@ for MEMDIR in "${MEMDIR_ARRAY[@]}"; do
         ;;
       "025" )
         for nn in $(seq 1 3); do
-          src="${BASE_CPLIC}/${CPL_OCNIC}/${PDY}${cyc}/${MEMDIR}/ocean/${PDY}.${cyc}0000.MOM.res_${nn}.nc"
+          src="${BASE_CPLIC}/${CPL_OCNIC:-}/${PDY}${cyc}/${MEMDIR}/ocean/${PDY}.${cyc}0000.MOM.res_${nn}.nc"
           tgt="${COM_OCEAN_RESTART}/${PDY}.${cyc}0000.MOM.res_${nn}.nc"
```

Comment:
@aerorahul should this be COM_OCEAN_RESTART_PREV?   I'm getting an error: 
`scripts/exglobal_stage_ic.sh: line 93: COM_OCEAN_RESTART: unbound variable`

Reply:
It absolutely should be.  
Apologies for missing this (and the low res test also missed this). 
I will issue a hotfix asap.

Reply:
Thanks for making the hotfix!  I can confirm that this change worked.  The forecast wasn't successful, but that was for other reasons - not this :)

Reply:
Fix is in #2070